closures.md

```diff
@@ -167,6 +168,76 @@ The implementation are the private variables `name`, `age` and the code inside t
 
```

Comment:
This example has potential but I feel like it needs a little framing. i.e. how encapsulation relates to factory functions...might be a little bit abstract at this point IMO 
> In JS, a very common pattern is to create a function that will return individual objects.  Let's look at an example of a *factory* function that will allow us to create objects representing individual people.